app.js

```diff
@@ -1,9 +1,12 @@
 const express = require("express");
 const apiRouter = require("./api-router");
+const {  handleServerErrors, handleNotFound } = require("./errors");
 const app = express();
 
 app.use('/api', apiRouter)
 app.use(express.json());
```

Comment:
This line of code shouldn't be in your repo at all yet. You're not using it, so it shouldn't be there 👍